fixtures/v2/webhooks/domain.lock/example.http

```diff
@@ -0,0 +1,9 @@
+POST / HTTP/1.1
+Host: example.com
+Accept-Encoding: gzip
+Content-Type: application/json
+User-Agent: DNSimple-Webhook-Notifier/8fd98a84516b72d5b863b27f93fcbaef61f06c03
+Content-Length: 643
+Connection: keep-alive
+
+{"data": {"domain": {"id": 1, "name": "example.com", "state": "registered", "account_id": 1010, "auto_renew": false, "created_at": "2023-03-02T02:39:18Z", "expires_at": "2024-03-02T02:39:22Z", "expires_on": "2024-03-02", "updated_at": "2023-08-31T06:46:48Z", "unicode_name": "example.com", "private_whois": false, "registrant_id": 101}}, "name": "domain.lock", "actor": {"id": "1010", "entity": "account", "pretty": "xxxxxxx-xxxxxxx-xxxxxxx@xxxxx.com"}, "account": {"id": 1010, "display": "xxxxxxx-xxxxxxx-xxxxxxx", "identifier": "xxxxxxx-xxxxxxx-xxxxxxx@xxxxx.com"}, "api_version": "v2", "request_identifier": "0f31483c-c303-497b-8a88-2edb48aa111e"}
```

Comment:
**Note for the future:**

It's a bit awkward that the payload doesn't have any information regarding the lock status. Generally, the payload carries over details about the change.

As we currently can't include the lock status in the domain (for the performance reason we discussed), we would need to add a separate, specific node just for it. Given the payload name carries over the status, I'd leave it as it is.

Should we ever be able to sync the lock status locally in the domain object, we should definitely expose it here.

/cc @OleMchls for knowledge about the current limitations imposed by not storing the status locally